spec/lib/packet-spec.js

```diff
@@ -24,12 +24,13 @@ describe("Packet", function() {
         magic: 1669485411,
         options:
         {
-            dhcpMessageType: { value: 3, name: 'DHCPREQUEST' },
+            dhcpMessageType: 0x2,
```

Comment:
Using that enum actually breaks the test with a "TypeError: value is out of bounds".